### ISSUE TITLE
Renaming test statistics to use the mu subscript more explicitly

### ISSUE THREAD

**Issue by kratsg** [scikit-hep/pyhf#1246]
Maybe too late to for such a change, but what do you think about using these names "q0", "qmu" and "qmu_tilde" in the hypotest `test_stat` argument? I believe they are more readable and the connection to the formulas in the asymptotics paper is even more clear with the "mu" subscript.

_Originally posted by @alexander-held in https://github.com/scikit-hep/pyhf/pull/1232#discussion_r553843992_

**Comment by lukasheinrich** [comment]
`qµ` `qµ~` `q0` or are unicode chars too much?

**Comment by alexander-held** [comment]
The unicode also looks good to me. The main motivation is that "qmu" and "qµ" read the same way as the variable in the paper,  but "q" does not.

**Comment by matthewfeickert** [comment]
> The unicode also looks good to me. The main motivation is that "qmu" and "qµ" read the same way as the variable in the paper, but "q" does not.

I agree with @alexander-held here w.r.t. trying to keep similarity with the paper names, though I think that if we're wanting to break with the paper and introduce more general notation that can be done if we want to give a lot of thought to it. Maybe something to save for an API breaking change in the future(?).

> `qµ` `qµ~` `q0` or are unicode chars too much?

I'm a bit curmudgeonly on using unicode a lot, but maybe I need to get over that. :P So while I don't think I would enthusiastically vote for it, I also wouldn't fight it.

**Comment by lukasheinrich** [comment]
i'm fine with not using unicode (or only as an alias).. i sharer the same concerns

**Comment by matthewfeickert** [comment]
> i'm fine with not using unicode (or only as an alias)

Oh I do like the idea of as an alias though. :+1:
